qiita_pet/test/test_portal.py

```diff
@@ -0,0 +1,58 @@
+from unittest import main
+from qiita_pet.test.tornado_test_base import TestHandlerBase
+
+from mock import Mock
+
+from qiita_db.user import User
+from qiita_pet.handlers.base_handlers import BaseHandler
+
+
+class TestPortal(TestHandlerBase):
+    def test_get(self):
+        BaseHandler.get_current_user = Mock(return_value=User("admin@foo.bar"))
+        response = self.get('/admin/portals/studies/')
+        self.assertEqual(response.code, 200)
```

Comment:
After chatting offline with @josenavas we agreed on adding a test for not empty body when reply is 200. Adding now but I want to discuss this in the next meeting (already in the list) because we should be doing something smarter than adding tests that test that there are specific words in the rendering.